{
    "config": {
        "abort": {
            "already_configured": "A szolg\u00e1ltat\u00e1s m\u00e1r konfigur\u00e1lva van",
            "no_services": "Nem tal\u00e1lhat\u00f3 szolg\u00e1ltat\u00e1s a v\u00e9gponton"
        },
        "error": {
            "cannot_connect": "Sikertelen csatlakoz\u00e1s"
        },
        "step": {
            "hassio_confirm": {
                "description": "Be k\u00edv\u00e1nja konfigur\u00e1lni a Home Assistantot, hogy csatlakozzon a b\u0151v\u00edtm\u00e9ny \u00e1ltal biztos\u00edtott Wyoming szolg\u00e1ltat\u00e1shoz: {addon}?"
            },
            "user": {
                "data": {
                    "host": "C\u00edm",
                    "port": "Port"
                }
            }
        }
    }
}